test/e2e/serial/tests/configuration.go

```diff
@@ -693,9 +697,195 @@ var _ = Describe("[serial][disruptive] numaresources configuration management",
+
+			kcList := &machineconfigv1.KubeletConfigList{}
+			err = fxt.Client.List(context.TODO(), kcList)
+			Expect(err).ToNot(HaveOccurred())
+
+			var targetedKC *machineconfigv1.KubeletConfig
+			for _, mcp := range mcps {
+				for i := 0; i < len(kcList.Items); i++ {
+					kc := &kcList.Items[i]
+					kcMcpSel, err := metav1.LabelSelectorAsSelector(kc.Spec.MachineConfigPoolSelector)
+					Expect(err).ToNot(HaveOccurred())
+
+					if kcMcpSel.Matches(labels.Set(mcp.Labels)) {
+						targetedKC = kc
+					}
+				}
+			}
+			Expect(targetedKC).ToNot(BeNil(), "there should be at least one kubeletconfig.machineconfiguration object")
+
+			kcObj, err := kubeletconfig.MCOKubeletConfToKubeletConf(targetedKC)
+			Expect(err).ToNot(HaveOccurred())
+			initialTopologyManagerPolicy := kcObj.TopologyManagerPolicy
+
+			By("modifying Topology Manager Policy under kubeletconfig to none")
+			Eventually(func(g Gomega) {
+				err = fxt.Client.Get(context.TODO(), client.ObjectKeyFromObject(targetedKC), targetedKC)
+				Expect(err).ToNot(HaveOccurred())
+
+				kcObj, err = kubeletconfig.MCOKubeletConfToKubeletConf(targetedKC)
+				Expect(err).ToNot(HaveOccurred())
+
+				applyNewTopologyManagerPolicy(&kcObj.TopologyManagerPolicy, v1beta1.NoneTopologyManagerPolicy)
+				err = kubeletconfig.KubeletConfToMCKubeletConf(kcObj, targetedKC)
+				Expect(err).ToNot(HaveOccurred())
+
+				err = fxt.Client.Update(context.TODO(), targetedKC)
+				g.Expect(err).ToNot(HaveOccurred())
+			}).WithTimeout(10 * time.Minute).WithPolling(30 * time.Second).Should(Succeed())
+			waitForMcpsCondition(fxt.Client, context.TODO(), mcps, machineconfigv1.MachineConfigPoolUpdated)
+
+			defer func() {
+				By("reverting kubeletconfig changes")
+				kcObj.TopologyManagerPolicy = initialTopologyManagerPolicy
+
+				err = fxt.Client.Get(context.TODO(), client.ObjectKeyFromObject(targetedKC), targetedKC)
+				Expect(err).ToNot(HaveOccurred())
+
+				kcObj, err = kubeletconfig.MCOKubeletConfToKubeletConf(targetedKC)
+				Expect(err).ToNot(HaveOccurred())
+
+				if kcObj.TopologyManagerPolicy == initialTopologyManagerPolicy {
+					return
+				}
+
+				Eventually(func(g Gomega) {
+					err = kubeletconfig.KubeletConfToMCKubeletConf(kcObj, targetedKC)
+					Expect(err).ToNot(HaveOccurred())
+
+					applyNewTopologyManagerPolicy(&kcObj.TopologyManagerPolicy, initialTopologyManagerPolicy)
+
+					err = fxt.Client.Update(context.TODO(), targetedKC)
+					g.Expect(err).ToNot(HaveOccurred())
+				}).WithTimeout(10 * time.Minute).WithPolling(30 * time.Second).Should(Succeed())
+
+				By("waiting for mcp to start updating")
+				waitForMcpsCondition(fxt.Client, context.TODO(), mcps, machineconfigv1.MachineConfigPoolUpdating)
+
+				By("wait for mcp to get updated")
+				waitForMcpsCondition(fxt.Client, context.TODO(), mcps, machineconfigv1.MachineConfigPoolUpdated)
+			}()
+
+			var schedulerName string
+			nroSchedObj := &nropv1.NUMAResourcesScheduler{}
```

Comment:
defining the type `var nroSchedObj &nropv1.NUMAResourcesSchedule` can be also be enough

Reply:
yes fixed